### ISSUE TITLE
Please DeviceSuport version 14.7?

### ISSUE THREAD

**Issue by CharlesPereira1** [iGhibli/iOS-DeviceSupport#160]
already have support for this version?

![image](https://user-images.githubusercontent.com/54192694/126237515-d34c4b98-2b01-480e-98ba-6ee0f655eee6.png)

**Comment by xscorpionxz** [comment]
looking for this version too

**Comment by merktassel** [comment]
As a workaround in the meantime, I was able to deploy apps using Xcode 12.4 (12D4e) to an iPhone running iOS 14.7 by copying the iOS 14.6 device support files and placing them in a folder named iOS 14.7. Navigate to `/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/DeviceSupport/`, copy the `14.6` folder, and rename it `14.7`. Be sure to restart Xcode afterwards. You can verify whether this worked by checking your device's status in the `Window > Devices and Simulators` menu. 

 Hope this helps.

**Comment by xscorpionxz** [comment]
Thanks alot for the quick solution !

**Comment by magic3584** [comment]
@merktassel Thanks guy

**Comment by magic3584** [comment]
@merktassel 
Is there a situation that Xcode lost device connection after a breakpoint?

**Comment by lizhili42** [comment]
> @merktassel
> Is there a situation that Xcode lost device connection after a breakpoint?

Yes, I would when I use Xcode12.4, 12.5.1 fix.

**Comment by merktassel** [comment]
> @merktassel
> Is there a situation that Xcode lost device connection after a breakpoint?

Yes, unfortunately debugging with breakpoints does not seem to function when using this workaround.

**Comment by sirnacnud** [comment]
> As a workaround in the meantime, I was able to deploy apps using Xcode 12.4 (12D4e) to an iPhone running iOS 14.7 by copying the iOS 14.6 device support files and placing them in a folder named iOS 14.7. Navigate to `/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/DeviceSupport/`, copy the `14.6` folder, and rename it `14.7`. Be sure to restart Xcode afterwards. You can verify whether this worked by checking your device's status in the `Window > Devices and Simulators` menu.
> 
> Hope this helps.

This didn't work for me with iOS 14.7.1 and Xcode 12.4. I had to get the 14.7 support files from https://github.com/JinjunHan/iOSDeviceSupport/tree/master/DeviceSupport

**Comment by LunightLab** [comment]
Thank you~ :)

**Comment by abdulrahim46** [comment]
> > As a workaround in the meantime, I was able to deploy apps using Xcode 12.4 (12D4e) to an iPhone running iOS 14.7 by copying the iOS 14.6 device support files and placing them in a folder named iOS 14.7. Navigate to `/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/DeviceSupport/`, copy the `14.6` folder, and rename it `14.7`. Be sure to restart Xcode afterwards. You can verify whether this worked by checking your device's status in the `Window > Devices and Simulators` menu.
> > Hope this helps.
> 
> This didn't work for me with iOS 14.7.1 and Xcode 12.4. I had to get the 14.7 support files from https://github.com/JinjunHan/iOSDeviceSupport/tree/master/DeviceSupport

Thank you!
